src/mocsolver.jl

```diff
@@ -336,18 +421,23 @@ function add_q_to_φ!(sol::MoCSolution, prob::MoCProblem)
     @inbounds for i in 1:NRegions
         xs = getxs(prob, i)
         @unpack Σt = xs
-
+        if volumes[i] < 1e-12 # deal with zero volume FSRs
+            volumes[i] = 1e30
+        end
```

Comment:
If volume is too small, set it to infinity? I don't understand this part.

Reply:
This was based on these few lines from OpenMOC, but it may be uneccessary: https://github.com/mit-crpg/OpenMOC/blob/7c8c9460c1c95f68dae102a402a39afa233a0b8c/src/CPUSolver.cpp#LL2621C1-L2623C21